velocidi-sdk/src/main/kotlin/com/velocidi/events/TrackingEvent.kt

```diff
@@ -16,10 +12,39 @@ abstract class TrackingEvent(
     /**
      * Serializes the current data model to JSON
      */
-    open fun serialize(): String {
+    open fun toJson(): String {
```

Comment:
Why do we still need this?

Reply:
In my opinion, this method gives more flexibility to SDK users. E.g., they can create a pageView, convert it to JSON and add custom properties and convert it back to a `CustomTrackingEvent`.

I don't have a strong opinion about this. If you think it's best to be stricter on this matter, I'll remove it.

Reply:
If that is the use case, can't we just provide a method to append custom properties directly from a `CustomTrackingEvent`?

Reply:
I removed the toJson operation in da4ca9c. I also added the possibility to append proprieties to customTrackingEvents in 932e100.